### PR TITLE
Added: Swipe module, Api connected to retrieve an array of movies 

### PR DESCRIPTION
**Pull request by johnandblue**

### CODE REVIEW

File: App/actions/actions.js [hunk 1]
```diff
@@ -1,86 +1,24 @@
-import { CALL_API } from '../lib/api'
+export const ADD_MOVIES = 'addMovies';
 
-export const getMovies = () => ({
-    type: 'GET_MOVIES',
-    [CALL_API]: {
-      endpoint: '/movies'
-    }
-  })
+// export const addMovies = (movies) => ({
```
Comment (arol) [inline App/actions/actions.js]: Unused code should be deleted.

File: App/reducers/reducers.js [hunk 2]
```diff
@@ -1,69 +1,90 @@
...
-    //   return Object.assign({}, state)
-    //   break;
-    // case 'DELETE_ALL_REQUEST':
-    // case 'DELETE_ALL_SUCCESS':
-    // return Object.assign({}, state, {loading: true})
-    // break;
-    case 'LOGOUT':
-      return [];
-    default:
+    // case 'ADD_MOVIES':
```
Comment (arol) [inline App/reducers/reducers.js]: The same as before. Unused code should be deleted.

File: App/reducers/reducers.js [hunk 3]
```diff
@@ -1,69 +1,90 @@
...
 
-const flag = (state = true, action) => {
-  switch (action.type) {
-    case 'CHANGE_FLAG':
-      console.log('in the reducer flag');
-      return !state;
-    default:
-  }
-  return state;
+// const user = (state = {}, action) => {
```
Comment (arol) [inline App/reducers/reducers.js]: Same as above. Keep it clean, "saving it for later" is not a reason to keep this in here.

File: app/actions/actions_template.js [hunk 4]
```diff
@@ -0,0 +1,86 @@
+// import { CALL_API } from '../lib/api'
```
Comment (arol) [inline app/actions/actions_template.js]: Same.

File: app/containers/AppContainer.js [hunk 5]
```diff
@@ -19,17 +15,15 @@ import {
   Button
 } from 'react-native';
 
+const path = 'https://movied.herokuapp.com/popular';
```
Comment (arol) [inline app/containers/AppContainer.js]: Unused constant? This shouldn't be here. Try to put this on the actions or the middleware.

**Review by arol — CHANGES_REQUESTED** [review]

File: App/actions/actions.js (hunk 1 above)
Comment (johnandblue) [reply App/actions/actions.js]: @arol Changes made. Deleted unused code

Comment (johnandblue) [inline App/actions/actions.js]: Changes made. Deleted unused code

File: App/reducers/reducers.js (hunk 2 above)
Comment (johnandblue) [reply App/reducers/reducers.js]: Changes made. Deleted unused code

File: App/reducers/reducers.js (hunk 3 above)
Comment (johnandblue) [reply App/reducers/reducers.js]: Done.

File: app/containers/AppContainer.js (hunk 5 above)
Comment (johnandblue) [reply app/containers/AppContainer.js]: Moved to a folder used for constants like Path for popular movies, or the path for the poster.

File: app/actions/actions_template.js (hunk 4 above)
Comment (johnandblue) [reply app/actions/actions_template.js]: Done

**Comment by arol** [comment]
👍

**Review by arol — APPROVED** [review]

**Pull request merged**